backend/api/v1/endpoints/auth.py

```
@@ -24,59 +22,67 @@
-
-        # Return the user's id
-        return {"id": user.id, "username": user.username}
         
-
+        return user
+    
     except jwt.ExpiredSignatureError:
-
         raise HTTPException(status_code=401, detail="Token expired")
+    except jwt.PyJWTError as e:
+        print(f"JWT error: {str(e)}")  # Log the specific error for further insight
```

Ruff (0.8.2)

Use explicit conversion flag

Replace with conversion flag

(RUF010)